### DISCUSSION TITLE
Feedback :  More context about community struggles & plans to solve them for next GCE cohort.

### DISCUSSION THREAD

**Discussion by parth721** [community/community#113387] [GitHub Education] [answered]
### Hi everyone,

Can you see any bridge, what the GCE need from my side and what I'm trying out ?

I applied for GCE feb cohort. They suggested to share more community details next time. But my question is that. When I join the college for engineering. It don't even have a coding club. Later that year we took permission from higher authorities and opened a coding club.Most of ou work is offine because we are gather and people we are intersted in development and opensource. We conduct 2 hackations, few events, and 10+ workshops on college level (offline) (all posts available on college portal). 

Due to our remote location (minimal presence of IT industry, startup) People are not very much aware of all those thing( development) and we don't have any sort senior support who tralled on this path and the commute to those regions where we have those people its not cost effective. It's like we are in a pond, trying to build community and foster development environment by random iteration. 

I am trying for becoming a member of GCE so that I can connect them  to a world-wide spread  GCE community to bring more people and provide better environment.

**Comment by parth721** [comment]
Label : campus expert

**Comment by samus-aran** [comment, marked as answer]
Hi @parth721 

While we understand your desire to gain insight into the selection process, we regret to inform you that due to the high volume of applications we received, we are unable to provide specific individualized feedback to all applicants.

I've included a link to another reply that we share some of the things the team look for in the applicants for the next time:
* https://github.com/orgs/community/discussions/113331#discussioncomment-8907172

**Reply by parth721** [reply]
Hi @samus-aran can you help to figure out an issue (my assumption you noticed student make some mistake without knowing) It has been 1.5yr I started my programming journey.
I tried a lots of fields, computer vision, devops, web. None of them are at my finger tip but the fear of their complexity is gone. 
My thoughts was explore different tech. in first 2yrs then work to gain proficiency.

Do you think its a good approach to becomes that kind of developers community/industry need? (People often ask me about it)

With GPT another question comes to my mind "how to help people to differentiate that whether they are using gpt for learning not for plagiarism"?

Thank you 🙂

**Reply by samus-aran** [reply]
> I tried a lots of fields, computer vision, devops, web. None of them are at my finger tip but the fear of their complexity is gone.
My thoughts was explore different tech. in first 2yrs then work to gain proficiency.

I think exploring a range of technologies through the first few years is a great way to identify patterns and similarities between them that builds up your experience to abstract the technology from the methodology. What I found useful at the beginning of my career was try out seemly opposite types of technology and languages, begin to identify their similarities and best practices, then narrow down to a couple of languages I liked because they were fun but reliable, then further narrow down to frameworks in that languages. 

The first few years of your learning journey is exactly that: learning. Learning what you like, what you feel confident using in your day to day (because remember you will be do this daily!) and learning how it all fits into the larger industry. I know it can be overwhelming with such a selection of areas but if you really enjoy it and find purpose in it then explore it - most will always have a need. You don't need to pick one and stick to it for the rest of your career.

> With GPT another question comes to my mind "how to help people to differentiate that whether they are using gpt for learning not for plagiarism"?

I'm not sure I understand this question. The differentiate to me would be that if the user is using GPT for learning then the outcome of that learning is their own creation by combination the knowledge they have accumulated. If a user is copying and pasting from GPT in isolation and passing it off as their own work they'll lack the understanding of the code and any alterations or future development may lead to poor code execution. But this difference has been in existence before GTP - I've been guilty of copy-paste code before to only discover that the short cut wasn't a short cut in the long run. 🤷
